frontend/src/app/api/deepseek/route.ts

```
@@ -0,0 +1,91 @@
+    const { econ, dipl, govt, scty } = req.body;
+    
+    // Validate required fields
+    if (!econ || !dipl || !govt || !scty) {
+      return res.status(400).json({ error: 'Missing required fields' });
+    }
+    
+    // Validate score ranges
+    const scores = { econ, dipl, govt, scty };
+    for (const [key, value] of Object.entries(scores)) {
+      const score = Number(value);
+      if (isNaN(score) || score < 0 || score > 100) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)